src/core/tools/networkToolFactory.js

```diff
@@ -0,0 +1,18 @@
+/*
+Copyright 2019 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+/**
+ * Tool-specific dependencies => config => componentCreator => result
+ */
+export default (createNetwork, lifecycle, logger) => config => () => {
+  return createNetwork(config, logger, lifecycle);
```

Comment:
This creates a new network instance for each component even though it doesn't depend on the componentCreator.  What is your reasoning?

Reply:
Ooh, good catch. No reasoning. Just a mistake. I'll fix it.